src/components/scoresheets/my_scoresheets_container.jsx

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import Scoresheets from './scoresheets';
+import { fetchScoresheet, fetchScoresheets, removeScoresheet } from '../../actions/scoresheet_actions';
+
+const mapStateToProps = (state) => {
+  return {
+    scoresheets: state.scoresheets,
```

Comment:
I think this is not ideal to just get ALL scoresheets. Instead, let's get an array of the user's scoresheets. I've added the array of scoresheet_ids to the current user backend. That will also save you from having to do all the awkward Object.values stuff in the component.

Reply:
Done!